README.md

```diff
@@ -22,19 +22,32 @@ Below are some common usage examples:
 
 ```shell
 # Build images of the hello-world containers
-bake -p hello-world -i 0.0.0-dev
+bake --product hello-world
 
 # Build only one version [0.37.2] of OPA
-bake -p opa=0.37.2 -i 0.0.0-dev
+bake --product opa=0.37.2
+
+# Dry run. Do not buikd anything. Print the the generated Bakefile.
+bake --product hello-world --dry
+
+# Build all OPA images and set the organisation to "sandbox"
+bake --product opa=0.37.2 --organization sandbox
+
+# Build all OPA images and set the image version to a release 24.7.0
+bake --product opa --image-version 24.7.0
 
 # Enable distributed docker cache (requires credentials to access the cache registry)
-bake -p opa --cache
+bake --product opa --cache
```

Comment:
Do we document anywhere where/how to provide those credentials?

Reply:
I'll add section to the readme

Reply:
Done https://github.com/stackabletech/image-tools/pull/38/commits/f7063c5027b88c3c07385320d9123f7e4594bec0